packages/dependency-finder/src/lib/find-missing.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { resolve, basename, dirname } from 'path';
+import { existsSync } from 'fs';
+
+type FoundPackages = {
+	foundPackages: Array< string >;
+	resolvedFiles: Array< string >;
+};
+
+/**
+ * Resolves dependencies which dependency-tree has trouble finding. For example,
+ * it is unable to parse sass imports such as `~@wordpress/base-styles/mixins`.
```

Comment:
> Using ~ is deprecated and can be removed from your code (we recommend it)

(TIL)

From https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules

Reply:
Oh nice! If that's the case, we might be able to remove this altogether if we can fix those. I dug into the source code of the dependency-tree library, and I'm not sure if it can handle SASS imports from node_modules. (I was originally going to file an upstream fix, but it was more tricky than it seemed, so settled for the current approach as a workaround.)

Reply:
PR: #52729. Unfortunately, we might only be able to do it for things using `calypso-build` to start (so excluding client)